src/main/java/net/dv8tion/jda/core/JDABuilder.java

```diff
@@ -50,6 +50,7 @@
     protected final List<Object> listeners;
 
     protected SessionReconnectQueue reconnectQueue = null;
+    protected ShardedRateLimiter shardRateLimiter = new ShardedRateLimiter();
```

Comment:
I would 
1) add a check to ShardedRateLimiter to prevent usage across multiple accounts 
2) initialize this lazily (like httpClientBuilder) to prevent having the same ShardedRateLimiter across multiple accounts when using the same JDABuilder

Reply:
I could set it when the setToken method is invoked